x/auth/module.go

```diff
@@ -151,6 +151,10 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(types.ModuleName, 4, func(ctx sdk.Context) error { return nil }); err != nil {
 		panic(fmt.Sprintf("failed to migrate x/%s from version 4 to 5: %v", types.ModuleName, err))
 	}
+
+	if err := cfg.RegisterMigration(types.ModuleName, 5, m.Migrate4To5); err != nil {
```

Comment:
```suggestion
	if err := cfg.RegisterMigration(types.ModuleName, 4, m.Migrate4To5); err != nil {
```

The comment below should be updated